### PR TITLE
FIXED THE MOST DANGEROUS BUG !!!!!!!!!!!!!!

### PR DESCRIPTION
**Pull request by not-lum**
mrg plz

### CODE REVIEW

File: crownbox.nimble [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Package
 
 version       = "0.1.0"
-author        = "IDF"
+author        = "lum"
```
Comment (ghost) [inline crownbox.nimble:4]: Bad

**Review by ghost — CHANGES_REQUESTED** [review]

File: crownbox.nimble (hunk 1 above)
Comment (leventea) [reply crownbox.nimble:4]: good

Comment (not-lum) [reply crownbox.nimble:4]: if u dont like it u dont like it

**Comment by ardek66** [comment]
Closing this PR as the world is not ready yet for such amazing work

**Pull request closed without merging**